Gemfile

```diff
@@ -8,7 +8,7 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 3.6.2"
+gem "jekyll", "~> 3.7"
```

Comment:
should this be 3.7 or 3.8?

Reply:
I suppose a better version pin would be `>= 3.7`. The fix is in 3.7, so as long as the pin upgrades us past that we're good.